GetFed/GetFed/Controllers/FoodEntryViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  FoodEntryViewController.swift
+//  GetFed
+//
+//  Created by Britney Smith on 12/4/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class FoodEntryViewController: UIViewController {
+    
+    // MARK - Outlets
+    @IBOutlet var foodTextField: UITextField!
+    @IBOutlet var brandTextField: UITextField!
+    @IBOutlet var proteinTextField: UITextField!
+    @IBOutlet var carbsTextField: UITextField!
+    @IBOutlet var fatTextField: UITextField!
+    @IBOutlet var foodLabel: UILabel!
+    @IBOutlet var brandLabel: UILabel!
+    @IBOutlet var proteinLabel: UILabel!
+    @IBOutlet var carbsLabel: UILabel!
+    @IBOutlet var fatLabel: UILabel!
+    @IBOutlet var scrollView: UIScrollView!
+    
+    // MARK - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        scrollView.contentSize = CGSize(width: self.view.frame.width, height: self.view.frame.height + 100)
+        setTextFieldPlaceholderText()
+        setLabelText()
+        setKeyboardTypes()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.isNavigationBarHidden = false
+    }
+    
+    // MARK - Methods
+    func setTextFieldPlaceholderText() {
+        foodTextField.placeholder = "ex: \"cookies\""
+        brandTextField.placeholder = "ex: \"Cookies Inc\""
+        proteinTextField.placeholder = "ex: \"5\""
+        carbsTextField.placeholder = "ex: \"35\""
+        fatTextField.placeholder = "ex: \"15\""
+    }
+    
+    func setLabelText() {
+        foodLabel.text = "Food Name:"
+        brandLabel.text = "Brand Name:"
+        proteinLabel.text = "Protein amount in grams (per 100 grams)"
+        carbsLabel.text = "Carbs amount in grams (per 100 grams)"
+        fatLabel.text = "Fat amount in grams (per 100 grams)"
+    }
+    
+    func setKeyboardTypes() {
+        proteinTextField.keyboardType = .decimalPad
+        carbsTextField.keyboardType = .decimalPad
+        fatTextField.keyboardType = .decimalPad
+    }
+
+    // MARK - Actions
+    @IBAction func cancelButtonTapped(_ sender: UIButton) {
```

Comment:
It's better (and Apple recommended practice) to name your actions after what they do rather than what button was tapped.

Reply:
This article really helped convince me: https://blog.cocoafrog.de/2018/04/12/How-to-name-IBActions.html

IBAction names refactored to `cancel` and `save`.